CmisSync.Lib/Sync/SynchronizedFolder.cs

```diff
@@ -1166,20 +1177,13 @@ private bool DownloadFile(IDocument remoteDocument, string localFolder)
                         }
                         else // No conflict
                         {
-                            // Should the local file be made read-only?
```

Comment:
Any special reason for removing this important addition made a few days ago? Probably a Git manipulation error? Could you please recreate your fork, apply your changes manually, then send a new pull request? Thanks a lot!